DigitalLearningSolutions.Web/ViewModels/SuperAdmin/Faqs/SearchableFaqViewModel.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.SuperAdmin.Faqs
+{
+    using System.Linq;
+    using DigitalLearningSolutions.Data.Enums;
+    using DigitalLearningSolutions.Web.Models.Enums;
+    using DigitalLearningSolutions.Web.ViewModels.Common.SearchablePage;
+
+    public class SearchableFaqViewModel : BaseFilterableViewModel
+    {
+        public SearchableFaqViewModel(SearchableFaqModel faq)
+        {
+            FaqId = faq.FaqId;
+            AHtml = faq.AHtml;
+            CreatedDate = faq.CreatedDate.ToShortDateString();
+            Published = faq.Published ? "Published" : "Unpublished";
+            QAnchor = faq.QAnchor;
+            QText = faq.QText;
+            TargetGroup = Enumeration.GetAll<DlsSubApplication>()
```

Comment:
I'm not sure this method of getting a target group will work across all FAQ target groups. There is not a corresponding `DlsSubApplication` for every `TargetGroup` exactly.

There is also no reference data in the DB.

It may be that we go with a simple `enum` containing the 4 TargetGroups with a simple description on each that can be pulled out. Or maybe use the more comprehensive `enumeration` class if that is easier.

Reply:
Ok cool so shall I create a new enum with just the data we need from the first 4 sub applications? Shall I also make it nullable so if it gives us something else we say null or maybe 'n/a'

Reply:
Oh I've just had a closer look 2 of the sub applications have target group ids but 1,3 dont seem related to a sub application